From 8cd0386a6076c83db851b01317d63eed4f847e98 Mon Sep 17 00:00:00 2001
From: Zhang Jian <zhangjian210@huawei.com>
Date: Thu, 30 Dec 2021 16:26:08 +0800
Subject: [PATCH] arm64/ascend: Enable CONFIG_ASCEND_OOM for
 openeuler_defconfig
Patch-mainline: Not yet, from openEuler
References: bsn#22
openEuler-commit: 8cd0386a6076c83db851b01317d63eed4f847e98


ascend inclusion
category: feature
bugzilla: https://gitee.com/openeuler/kernel/issues/I4K2U5
CVE: NA

-------------------------------------------------

Enable the ascend oom control features for openeuler_defconfig default config.

Signed-off-by: Zhang Jian <zhangjian210@huawei.com>
Reviewed-by: Kefeng Wang <wangkefeng.wang@huawei.com>
Signed-off-by: Zheng Zengkai <zhengzengkai@huawei.com>
Signed-off-by: Guoqing Jiang <guoqing.jiang@suse.com>
---
 arch/arm64/configs/openeuler_defconfig | 1 +
 1 file changed, 1 insertion(+)

diff --git a/arch/arm64/configs/openeuler_defconfig b/arch/arm64/configs/openeuler_defconfig
index 6cdcbad3967e..fb5b9650b458 100644
--- a/arch/arm64/configs/openeuler_defconfig
+++ b/arch/arm64/configs/openeuler_defconfig
@@ -476,6 +476,7 @@ CONFIG_RANDOMIZE_MODULE_REGION_FULL=y
 CONFIG_ASCEND_FEATURES=y
 CONFIG_ASCEND_DVPP_MMAP=y
 CONFIG_ASCEND_CLEAN_CDM=y
+CONFIG_ASCEND_OOM=y
 # end of Kernel Features
 
 #
-- 
2.26.2

